charactersheet/charactersheet/viewmodels/root.js

```diff
@@ -127,6 +127,49 @@ function RootViewModel() {
 		self.activeTab('players');
 	};
 
+	//Tab shortcut Methods
```

Comment:
It looks like this PR is trying to merge in an old version of the keyboard shortcuts. @tanyxp could you remove this before I merge?